code/modules/power/cell.dm

```diff
@@ -118,9 +118,7 @@
 		rigged = 1 //broken batterys are dangerous
 
 /obj/item/weapon/cell/emp_act(severity)
-	charge -= 1000 / severity
-	if (charge < 0)
-		charge = 0
+	charge = max(charge - maxcharge/severity, 0)
```

Comment:
Does it make sense that if you have two power cells with 1000 charge left, a normal cell loses less charge than a hyper cell?

That seems unintuitive. On the other hand, this is potentially a huge nerf to borgs so that's interesting.